### PR TITLE
LRAC-15075 follow up unit tests

### PR DESCRIPTION
**Pull request by fmarcio**
https://liferay.atlassian.net/browse/LRAC-15075

### CODE REVIEW

**Review by interaminense — APPROVED** [review]
LGTM! Thanks @fmarcio

**Comment by interaminense** [comment]
ci:forward

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/components/__tests__/__snapshots__/LineChart.jsx.snap [hunk 1]
```diff
@@ -1,6 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`LineChart should get short intervals 1`] = `<div />`;
+exports[`LineChart should get short intervals 1`] = `
```
Comment (fmarcio) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/components/__tests__/__snapshots__/LineChart.jsx.snap:3]: @interaminense I believe these snaps update are related to that Recharts update we had to do

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**